bukkit/build.gradle.kts

```diff
@@ -34,6 +34,7 @@ bukkit {
     apiVersion = "1.13"
     author = "4drian3d"
     load = net.minecrell.pluginyml.bukkit.BukkitPluginDescription.PluginLoadOrder.STARTUP
+    foliaSupported = true
```

Comment:
I plan to remove this module completely very soon, so this won't be necessary, but I'll leave it up anyway